migrations/0020-add-eventFeedback-table.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner, TableColumn } from 'typeorm';
+
+export class AddEventFeedbackTable1602120071159 implements MigrationInterface {
```

Comment:
Nit: not a table, `0021-add-event-feedback.ts` (21, because 20's already merged and we want to maintain linearity).